internal/config/profile.go

```diff
@@ -38,6 +38,7 @@ import (
 
 const (
 	ToolName                     = "mongocli"      // ToolName of the CLI
+	AtlasToolName                = "atlascli"      // AtlasToolName of the atlas CLI
```

Comment:
At this point I would set the tool name at build time

Reply:
updated